archetypes/swarm-greenfield-archetype/pom.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!--
+
+     Copyright 2005-2015 Red Hat, Inc.
+
+     Red Hat licenses this file to you under the Apache License, version
+     2.0 (the "License"); you may not use this file except in compliance
+     with the License.  You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+     implied.  See the License for the specific language governing
+     permissions and limitations under the License.
+
+--><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.obsidiantoaster.quickstarts</groupId>
+    <artifactId>archetypes</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>swarm-greenfield-archetype</artifactId>
+  <name>Fabric8 :: Archetypes :: swarm-greenfield</name>
+  <packaging>maven-archetype</packaging>
+
+  <description>Creates a new Obsidian :: Quickstart :: Wildfly Swarm Greenfield</description>
```

Comment:
Remove the` Obsidian :: Quickstart ::` prefix in your quickstart's pom.xml and regenerate it again. There is a JIRA asking to remove tha lt suffix from our quickstarts